daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -1351,6 +1351,11 @@ private[lf] object Speedy {
         machine.returnValue = cached.any
       }
     }
+
+  }
+
+  private[speedy] final case class KContinue(continue: SValue => Unit) extends Kont {
```

Comment:
I'm not quite sure if that super generic continuation is the right approach. You can subsume pretty much all other continuations under this so it’s not clear when this is used vs other stuff. Maybe make a continuation that is specific to checking visibility?